mars/services/subtask/worker/processor.py

```diff
@@ -489,6 +501,20 @@ async def run(self):
                 await self._unpin_data(input_keys)
 
         await self.done()
+        if self.result.status == SubtaskStatus.succeeded:
+            cost_time_secs = (
+                self.result.execution_end_time - self.result.execution_start_time
+            )
+            logger.info(
+                "Time consuming to execute a subtask is %ss with session_id %s, subtask_id %s",
```

Comment:
```suggestion
                "Executing subtask %s with session_id %s took %s",
```

Reply:
Good suggestion. I'll change it in later pr.